conf/route_control.py

```diff
@@ -296,6 +297,17 @@ def probe_addr(item, src_mac):
     arpcache[item.neighbor_ip] = item
     print('Adding entry {} in arp probe table'.format(item))
 
+    try:
+        ipb = ipaddress.ip_address(item.neighbor_ip)
+        print("The IP address {} is valid.".format(ipb))
+        if isinstance(ipb, ipaddress.IPv4Address):
+          print("The IP address {} is valid ipv4 address".format(ipb))
+        else:
+          print("The IP address {} is valid ipv6 address. Ignore ".format(ipb)
```

Comment:
Missing `)` at the end of the line... It should be:
```suggestion
          print("The IP address {} is valid ipv6 address. Ignore ".format(ipb))
```